src/NeoSharp.Core/Cryptography/ECPoint.cs

```diff
@@ -51,10 +51,22 @@ private ECPoint()
         /// <param name="point">Point</param>
         public ECPoint(byte[] point)
         {
-            IsInfinity = false;
+            if (point == null)
```

Comment:
this is on other PR.

Reply:
damm .... looks like I mistake the PR ... 
after merge the other I will solve this conflict.